uint32 ping_number
uint32 start_mm
uint32 length_mm
uint32 timestamp_ms
uint32 ping_hz
uint16 gain_index
uint16 num_results
uint16 sos_dmps
uint8 channel_number
uint8 reserved
float32 pulse_duration_sec
float32 analog_gain
float32 max_pwr_db
float32 min_pwr_db
float32 transducer_heading_deg
float32 vehicle_heading_deg
uint16[] pwr_results
